src/styles/_catalog.scss

```diff
@@ -1,22 +1,34 @@
 .catalog {
-  display: flex;
-  flex-wrap: wrap;
+  display: grid;
+  grid-template-columns: repeat(4, $cardWidth);
+  gap: 48px;
+  justify-content: center;
 
-  max-width: 944px;
-  margin: 0 auto;
+  @media (max-width: 1024px) {
+    & {
+      grid-template-columns: repeat(3, $cardWidth);
+    }
+  }
+
+  @media (max-width: 768px) {
+    & {
+      grid-template-columns: repeat(2, $cardWidth);
+    }
+  }
+
+  @media (max-width: 480px) {
+    & {
+      grid-template-columns: repeat(1, $cardWidth);
+    }
+  }
```

Comment:
https://mateacademy.slack.com/archives/C03H0MLSFUJ/p1655716733610009

Reply:
sorry, last commit did not download